platforms/r3-corda/charts/h2-addUser/Chart.yaml

```diff
@@ -6,6 +6,6 @@
 
 apiVersion: v1
 appVersion: "2.0"
-description: A Helm chart for registering the notary with the nms
-name: h2-pass-change
-version: '0.14.0'
+description: "R3-corda-os: Its authenticate with the Vault server for Corda Opensource"
```

Comment:
This chart is not for authenticating use. If this is never used from the roles or playbook, remove these charts